docs/source/api/plugin/usage-reporting.md

```diff
@@ -1,20 +1,22 @@
 ---
-title: "API Reference: ApolloServerPluginUsageReporting"
-sidebar_title: ApolloServerPluginUsageReporting
+title: "API Reference: Usage reporting plugin"
+sidebar_title: "Usage reporting plugin"
```

Comment:
Can you update the sidebar titles for the other plugins at the same time, even if you're not updating the actual docs in the same PR? It looks inconsistent otherwise.